doc/Caliptra.md

```diff
@@ -233,116 +233,116 @@ In accordance with OCP Attestation specification [Reference 8](#ref-8), devices
-|Side channel - Timing|Attacker observes the elapsed time of different sensitive operations.|Includes attacks where the attacker actively stimulates the sensitive operations while timing.|
-|Cryptographic Analysis|Attacker observes plaintext, ciphertext, related data, or immediate values in cryptography to overcome cryptographic controls|<p>Includes all practical cryptanalysis attacks.</p><p>Assumes NIST-unapproved algorithms provide no security. (e.g. SHA-1, Single DES, ChaCha20)</p><p>Assumes any cryptographic algorithm that provides less than 128 bits of security (as determined by NIST SP 800-57) provides no security.</p><p>Excludes quantum computer attacks. This exclusion will be removed soon.</p>|
+| Debug and register interfaces | Manipulation of externally accessible registers of Caliptra.|Includes all buses that are accessible to components external to Caliptra, including JTAG.|
+| Software interfaces       | Attacker invokes software interfaces that are exposed by Caliptra to external components. | <p>Includes all externally exposed software interfaces from both non-RoT firmware as well as interfaces accessed by external IP blocks.</p><p>Includes exploiting both design and implementation flaws.</p><p>For high value assets only (see next subsection), the attacker is assumed to fully control all mutable code of the SoC, including privileged Caliptra mutable code.</p> |
+|Side channel - timing|Attacker observes the elapsed time of different sensitive operations.|Includes attacks where the attacker actively stimulates the sensitive operations while timing.|
+|Cryptographic analysis|Attacker observes plaintext, ciphertext, related data, or immediate values in cryptography to overcome cryptographic controls.|<p>Includes all practical cryptanalysis attacks.</p><p>Assumes NIST-unapproved algorithms provide no security (for example, SHA-1, Single DES, ChaCha20).</p><p>Assumes any cryptographic algorithm that provides less than 128 bits of security (as determined by NIST SP 800-57) provides no security.</p><p>Excludes quantum computer attacks. This exclusion will be removed soon.</p>|
 
-### Trust Boundaries
+### Trust boundaries
 
-Following diagram establishes trust boundaries for the discussion of threat modeling. Caliptra based SoCs are expected to have Caliptra as silicon RoT, platform or SoC security engine to orchestrate SoC security needs and rest of the SoC.
+The following figure shows trust boundaries for the discussion of threat modeling. SOCs based on Caliptra are expected to have Caliptra as silicon RoT, and are expected to have a platform or SoC security engine to orchestrate SoC security needs and the rest of the SoC.
```

Comment:
s/SOC/SoC/

(I noticed there's also "SOC" up in the header, "Caliptra: A Datacenter System on a Chip (SOC) Root of Trust (RoT)")